diff --git a/node_modules/capacitor-live-activity/ios/Sources/LiveActivityPlugin/LiveActivity.swift b/node_modules/capacitor-live-activity/ios/Sources/LiveActivityPlugin/LiveActivity.swift
index f0a1c65..8eca7a0 100644
--- a/node_modules/capacitor-live-activity/ios/Sources/LiveActivityPlugin/LiveActivity.swift
+++ b/node_modules/capacitor-live-activity/ios/Sources/LiveActivityPlugin/LiveActivity.swift
@@ -19,7 +19,7 @@ private let EVT_ACTIVITY_UPDATE = "liveActivityUpdate"
                 let id = activity.attributes.id
 
                 switch activity.activityState {
-                case .active, .stale, .pending:
+                case .active, .stale:
                     activities[id] = activity
                 case .ended, .dismissed:
                     /// No action: ignore ended/dismissed activities (cleanup)
@@ -135,7 +135,7 @@ private let EVT_ACTIVITY_UPDATE = "liveActivityUpdate"
         Task { [weak self] in
             for await a in Activity<GenericAttributes>.activityUpdates {
                 switch a.activityState {
-                case .active, .stale, .pending:
+                case .active, .stale:
                     self?.activities[a.attributes.id] = a
                 case .ended, .dismissed:
                     self?.activities.removeValue(forKey: a.attributes.id)
